plugin/handler_socket/libhsclient/allocator.hpp

```diff
@@ -42,6 +43,19 @@ typedef std::allocator<int> allocator_type;
 	static_cast<typ *>(malloc((len) * sizeof(typ)))
 #define DENA_ALLOCA_FREE(x) free(x)
 #endif
+*/
+
+// AIX
+#if 1
+#define DENA_ALLOCA_ALLOCATE(typ, len) \
+        static_cast<typ *>((void*)alloca((len) * sizeof(typ)))
+#define DENA_ALLOCA_FREE(x)
+#else
+#define DENA_ALLOCA_ALLOCATE(typ, len) \
+        static_cast<typ *>((void*)malloc((len) * sizeof(typ)))
+#define DENA_ALLOCA_FREE(x) free(x)
+#endif
+
```

Comment:
alloca() returning char* looks ridiculous. I'm fine with cast for alloca, with minor preference to cast like:
``(typ*) alloca((len) * sizeof(typ))``

But please cleanup the rest.